pkg/cmd/server/api/v1/types_test.go

```diff
@@ -118,6 +118,8 @@ imageConfig:
   format: ""
   latest: false
 kind: MasterConfig
+kubeAdmissionControlPluginsOverride:
```

Comment:
The case of running openshift on top of a separate kube would need to be thought through. If we pass openshift-specific plugins to the kube config, the kube master won't find them and won't start.